curriculum/js3/3.js

```diff
@@ -4,10 +4,24 @@
  * @return {array} b
  */
 
-const solution = (a)=>{
-  return b;
-}
+const findDuplicates=(obj,keys=Object.keys(obj), values=Object.values(obj), i=0, newArr=[])=>{
+    if( i === values.length ) { return newArr };
+    if (values[i] === 2) {
```

Comment:
What if there are triples? Quadruples?

Reply:
question is only asking for duplicate numbers